clang/lib/DPCT/RulesAsm/Parser/AsmTokenKinds.def

```diff
@@ -278,6 +278,7 @@ STATE_SPACE(reg,    ".reg")
 STATE_SPACE(sreg,   ".sreg")
 STATE_SPACE(const,  ".const")
 STATE_SPACE(global, ".global")
+STATE_SPACE(volatile, ".volatile")
```

Comment:
According to the values in ss in https://docs.nvidia.com/cuda/parallel-thread-execution/#data-movement-and-conversion-instructions-ld,  .volatile is not a ss value, should not added here.

Reply:
Added as a modifier. As per docs it is a qualifier.